attestation-agent/attestation-agent/src/bin/ttrpc-aa-client.rs

```diff
@@ -101,15 +101,15 @@ pub async fn main() {
         ttrpc::asynchronous::Client::connect(&args.attestation_sock).expect("connect ttrpc socket");
     let client = AttestationAgentServiceClient::new(inner);
     match args.operation {
-        Operation::GetTee => {
-            let req = GetTeeTypeRequest {
+        Operation::GetTees => {
+            let req = GetTeeTypesRequest {
                 ..Default::default()
             };
             let res = client
-                .get_tee_type(context::with_timeout(TIMEOUT), &req)
+                .get_tee_types(context::with_timeout(TIMEOUT), &req)
                 .await
                 .expect("request to AA");
-            println!("{}", res.tee);
+            println!("{}", res.tees);
```

Comment:
Emmm. Still think about the hierarchical design. In the current fashion, it will say something like
```
tdx, nvidia-gpu, some-nic
```

in the hierarchical design, it will say (any name for the first term)
```
platform/cpu/basic: tdx+ccel+aael
devices: [nvidia-gpu, some-nic]
```
or
```
platform/cpu/basic: azure-vtpm-tdx(tdx+vtpm)
devices: [nvidia-gpu, some-nic]
```


wdyt?